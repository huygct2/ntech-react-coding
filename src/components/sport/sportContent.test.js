import React from 'react';
import renderer from 'react-test-renderer';
import SportContent from './sportContent';

test('SportContent Component', () => {
  const component = renderer.create(
    <SportContent
      propositions={[]}
      marketLength={20} />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
