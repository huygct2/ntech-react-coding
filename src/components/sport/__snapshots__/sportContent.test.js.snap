// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SportContent Component 1`] = `
<div
  className="sports has-text-centered"
>
  <div
    className="sports__header has-text-weight-semibold"
  >
    Straight
  </div>
  <div
    className="columns sports__bet"
  />
  <div
    className="sports__footer"
  >
    <div
      className="sports__all"
    >
      <a
        className="has-text-weight-bold is-uppercase"
      >
        View all markets (20)
      </a>
    </div>
    <span>
      Prices indicative until login & may change before bet is accepted
    </span>
  </div>
</div>
`;
